src/main/java/com/iexec/core/replicate/ReplicatesController.java

```diff
@@ -124,16 +124,17 @@ public ResponseEntity<TaskNotificationType> updateReplicateStatus(
                 return replicatesService
                         .updateReplicateStatus(chainTaskId, walletAddress, statusUpdate, updateReplicateStatusArgs)
                         .map(ResponseEntity::ok)
-                        .orElse(ResponseEntity.status(HttpStatus.FORBIDDEN.value())
-                                .build());
+                        .orElse(ResponseEntity.status(HttpStatus.FORBIDDEN)
+                                .body(TaskNotificationType.PLEASE_ABORT));
```

Comment:
Thank you very much for that, maybe add a note here about the why?

Reply:
I second that. It would be great to have to comment to remember later why we did this :)

Reply:
Does the last commit help ?

Reply:
See 57143be

Reply:
Perfect, thanks!